.github/workflows/pull-request.yml

```diff
@@ -38,8 +38,12 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
-      - name: Run prettier
-        run: pnpm run lint
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: latest
+      - name: Run Biome
+        run: biome ci .
```

Comment:
What is the difference between this and running `pnpm run lint`?

Reply:
This command is for use in CI https://biomejs.dev/reference/cli/#biome-ci

Reply:
Ah okay. That sounds like it runs a lot more than the equivalent commands in package.json? Those commands seem to just run the formatter, but the ci command runs the formatter + linter + sorter?

Reply:
`biome check .` runs formatter + linter + sorter [docs](https://biomejs.dev/reference/cli/#biome-check)
`biome ci .` runs linter + formatter in read-only [docs](https://biomejs.dev/reference/cli/#biome-ci)